pkg/controller/controlplane/valuesprovider.go

```diff
@@ -101,6 +101,7 @@ func shootAccessSecretsFunc(namespace string) []*gutil.ShootAccessSecret {
 		gutil.NewShootAccessSecret("csi-snapshotter", namespace),
 		gutil.NewShootAccessSecret("csi-resizer", namespace),
 		gutil.NewShootAccessSecret("csi-snapshot-controller", namespace),
+		gutil.NewShootAccessSecret(alicloud.CSISnapshotValidationName, namespace),
```

Comment:
For code consistency, maybe better use "csi-snapshot-validation" instead of alicloud.CSISnapshotValidationName.